static/app/components/contextPickerModal.tsx

```diff
@@ -429,13 +500,23 @@ class ContextPickerModalContainer extends Component<ContainerProps, ContainerSta
         organizations={this.state.organizations}
         organization={this.state.selectedOrganization!}
         onSelectOrganization={this.handleSelectOrganization}
+        configs={configs || []}
       />
     );
   }
 
   render() {
-    const {projectSlugs} = this.props;
+    const {projectSlugs, configUrl} = this.props;
 
+    if (configUrl && this.state.loading) {
+      return <LoadingIndicator />;
+    }
+    if (this.state.configs?.length) {
```

Comment:
@NisanthanNanthakumar what happens if there are 0 configs for that integration?

Reply:
@scefali its the default behaviour if 0 configs or not superuser. We do a check here:
https://github.com/getsentry/sentry/blob/353ad0fca1046b3f614a188d8e74a1eadd475714/static/app/components/contextPickerModal.tsx#L133
And that function is called in componentDidMount